server/__init__.py

```diff
@@ -82,14 +82,14 @@ def checkValue(infoList, filterParams, category, value):
 class TechJournal(Resource):
 
     def checkSubmission(self, filterParams, submission, searchObj, targetVal, category):
-        keyMatch = False
+        keyMatch = [False, -1]
```

Comment:
What's the reason for packing a `-1` in here? Doesn't seem like `keyMatch[1]` appears anywhere in the diff...